app/components/media_component.html.erb

```diff
@@ -83,13 +83,6 @@
         <%= render LockedStatusComponent.new %>
       </div>
 
-      <div data-controller="locked-audio-poster" data-action="auth-success@window->locked-audio-poster#hide needs-login@window->locked-audio-poster#show auth-denied@window->locked-audio-poster#show" role="presentation" style="flex: 1 0 100%;" hidden>
```

Comment:
We no longer use the audio tag for the viewer.

Reply:
Per https://github.com/sul-dlss/sul-embed/issues/2747#issuecomment-2701535741 I thought we were leaving the audio behavior as is?

Reply:
Yeah but we aren't actually using the <audio> tag anymore so audio is treated the same way, I am removing code that doesn't run anymore because the <audio> tag isn't used anymore. https://github.com/sul-dlss/sul-embed/pull/2386. So this code isn't doing anything.